backend/src/modules/Announcements/repository/inMemory/InMemoryAnnouncement.repository.js

```diff
@@ -0,0 +1,10 @@
+export class InMemoryAnnouncementRepository {
+  constructor() {
+    this.entities = {};
+  }
+
+  async addAnnouncement(newAnnouncement) {
+    this.entities[newAnnouncement.id] = newAnnouncement;
```

Comment:
Chyba tego nie widzę, w którym miejscu tworzysz to id (gdy nie masz połączenia z mongo)?

Reply:
Celna uwaga - gdzieś mi to też uciekło podczas CR

Reply:
wymyśliłem sobie taki trick: Przy tworzeniu obiektu Announcement w serwisie jeżeli nie przekazuję id to tworzy się przy pomocy uuid. Następnie jeżeli mamy mongo, to te id zostanie zastąpione id z mongo, a jeżeli mongo nie mamy to będziemy mieć id z uuid ;)